lib/features/family/features/profiles/cubit/profiles_state.dart

```diff
@@ -4,7 +4,9 @@ abstract class ProfilesState extends Equatable {
   const ProfilesState({
     required this.profiles,
     required this.activeProfileIndex,
+    this.cachedMembers = const [],
```

Comment:
I think in the PR you are only putting this in when calling the ProfilesUpdatedState. So I would not make it a member of ProfileState. Just of the ProfilesUpdatedState.

Reply:
I do use the data in ProfilesNotSetupState, and this comment revealed  a bug to me that i just fixed, thanks!